pkg/console/subresource/consoleserver/types.go

```diff
@@ -68,6 +68,7 @@ type ClusterInfo struct {
 	ControlPlaneToplogy configv1.TopologyMode `yaml:"controlPlaneTopology,omitempty"`
 	ReleaseVersion      string                `yaml:"releaseVersion,omitempty"`
 	NodeArchitectures   []string              `yaml:"nodeArchitectures,omitempty"`
+	CopiedCSVsDisabled  bool                  `yaml:"copiedCSVsDisabled,omitempty"`
```

Comment:
remove the omitempty here otherwise the yaml pkg whats Marshalling into yaml will drop the variable if its set to `false`

Reply:
I would prefer it drops the variable if it is false.  I set it back to `omitempty` which fixes the testcases and this prevents the config from showing `disableCopiedCSVs: false`

Reply:
Dont have any strong feelings either way. Lets keep it then.